### DISCUSSION TITLE
Which version will be the next LTS release of OpenSSL?

### DISCUSSION THREAD

**Discussion by huiyuexu** [openssl/openssl#23735] [Q&A] [answered]
[Time-based release Policy](https://www.openssl.org/policies/general/release-policy.html):

```
The OpenSSL release schedule follows a biannual model, with new versions being launched every April and October. 
```

[Release Strategy](https://www.openssl.org/policies/releasestrat.html)
```
LTS releases will be supported for at least five years and we will specify one at least every four years. 
```

According to the policy, 3.3 and 3.4 will released in 2024.

Which of the following is the next LTS version of OpenSSL? 3.3 or 3.4?

**Comment by mattcaswell** [comment, marked as answer]
The current LTS (3.0) was released on 7th September 2021. According to our current policy we must specify the next one within 4 years (i.e. by 7th September 2025). So, all of the next 3 releases are within that timescale (April 2024, October 2024, April 2025).

No decision has yet been made as to which release will be the next LTS.

**Comment by liudongmiao** [comment]
@mattcaswell Any chance to make a new LTS early for quic to support http3?

**Reply by ranisalt** [reply]
This would be really important for Node.js

**Comment by devhaozi** [comment]
3.4 is not a LTS version, apparently the next LTS will be 3.5 which will be released in April 2025.

**Reply by paulidale** [reply]
3.4 could still be the next LTS.  The latest version before the four year mark does not have to be made LTS.

**Comment by naynajain** [comment]
I have been curious.. is there a decision now made on which version would be LTS ? I don't see any update since last Oct 30th discussion.

**Reply by jericson** [reply]
Yes, [3.5 will be an LTS](https://openssl-library.org/post/2025-02-20-openssl-3.5-lts/).

**Reply by naynajain** [reply]
Thanks.

Thanks & Regards,
    - Nayna

**Comment by jericson** [comment]
> Which of the following is the next LTS version of OpenSSL? 3.3 or 3.4?

As it turns out, neither. The next LTS, coming in April, will be [3.5](https://openssl-library.org/post/2025-02-20-openssl-3.5-lts/).

Going forward the plan is to designate an LTS every two years. So the version coming in April, 2027 will also be an LTS that overlaps with 3.5.
